cmd/litestream/main.go

```diff
@@ -363,6 +366,12 @@ type ReplicaConfig struct {
 	Password string `yaml:"password"`
 	KeyPath  string `yaml:"key-path"`
 
+	// NATS settings
+	JWT            string `yaml:"jwt"`
+	Seed           string `yaml:"seed"`
```

Comment:
All auth methods should be supported. If an additional dependency is welcome, this package handles all the options as well as looking up the `nats context` profile. https://pkg.go.dev/github.com/nats-io/jsm.go/natscontext